### ISSUE TITLE
Pylance auto-imports in native notebooks don't work/have missing completion tooltip info

### ISSUE THREAD

**Issue by savannahostrowski** [microsoft/vscode-jupyter#5705]
## Environment data

-   VS Code version: 1.55.2
-   Jupyter Extension version (available under the Extensions sidebar): v2021.5.745244803
-   Python Extension version (available under the Extensions sidebar): v2021.4.765268190
-   OS (Windows | Mac | Linux distro) and version: Windows
-   Python and/or Anaconda version: 3.8.6
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): pyenv
-   Jupyter server running: Local

- Pylance extension version: v2021.4.3

## Expected behaviour
![image](https://user-images.githubusercontent.com/8949415/116614421-c3777200-a8ee-11eb-8cb9-ead1caba7ec7.png)

## Actual behaviour

![image](https://user-images.githubusercontent.com/8949415/116614332-a80c6700-a8ee-11eb-8ba1-fa223cbed616.png)

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. Type gc in a cell without `import gc`
2. See completion for gc is an auto-import (note the missing info in screenshots above)
2. Select it
3. Notice that the import statement is not added to the top of the cell/nothing happens

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

**Comment by rchiodo** [comment]
Auto import isn't possible in a notebook at the moment. We may need to remove this extra information.

**Comment by savannahostrowski** [comment]
Makes sense. Another option is to potentially prevent Pylance from displaying auto-imports in cells temporarily (and reenable them once the functionality is in place, if your users deem them valuable) under-the-hood.

At the user level, they can be disabled via `python.analysis.autoImportCompletions`: `false`.

**Comment by benlindsay** [comment]
#1686 was closed as a duplicate of this, and I'm interested in having Sort Imports available for the whole notebook. I'm not sure what the difference between organizing and sorting imports is. The `python.sortImports` command works on individual cells of a notebook, but not on the whole notebook. Should discussions about that be a separate issue from this one?

Thanks in advance!

**Comment by rchiodo** [comment]
@benlindsay I'd open an issue on [Pylance ](https://github.com/microsoft/pylance-release/issues) to implement sorting imports in notebooks. It's more complicated because where do the imports go? To the top cell? To the same cell? I think they'd likely want to discuss options.

**Comment by benlindsay** [comment]
I would think it would be a relatively straightforward choice to sort imports independently for each cell as if each one is a separate document, but yeah I'll go open an issue there if that's where it makes the most sense. Thanks!

**Comment by DonJayamanne** [comment]
Closing as fixed, i get an option to auto import and it works as expected
